mindmeld/components/request.py

```diff
@@ -352,12 +388,16 @@ class Request:
         default=attr.Factory(tuple), converter=tuple_elems_to_immutable_map
     )
 
-    def to_dict(self):
+    def to_dict(self) -> Dict:
```

Comment:
Minor gripe (since we aren't testing this yet with mypy), bare `Dict` would produce an error because it needs its constituent key/value types as well. This should be, at minimum (and fine for now),`Dict[str, Any]`. Better would be to use `Dict[str, SomeOtherType]`, where `SomeOtherType` encapsulates the value types we expect to find in the dictionary we return.

It looks like there are a few other examples in here as well that we should clean up.